crates/bevy_input/src/input.rs

```diff
@@ -20,6 +20,15 @@ use bevy_ecs::schedule::State;
 /// * [`Input::just_pressed`] will return `true` for one frame after a press event.
 /// * [`Input::just_released`] will return `true` for one frame after a release event.
 ///
+/// ## Button types
+///
+/// This crate provides support for the following button types:
+///
+/// * [`KeyCode`](crate::KeyCode)
+/// * [`ScanCode`](crate::ScanCode)
```

Comment:
```suggestion
/// * [`KeyCode`](crate::KeyCode) and [`ScanCode`](crate::ScanCode)
///   for keyboard input
```

I like these two together but it's not mandatory for me.

Reply:
Updated.